### ISSUE TITLE
Master build fail

### ISSUE THREAD

**Issue by YamStranger** [jcabi/jcabi-aspects#182]
@dmarkov As I understand from issue #173 test fail under 1.6 java, but it seems that master build fail under 1.7. 

Here is listing:

```
[INFO] [WARNING] diagnostic: warning: No SupportedSourceVersion annotation found on com.jcabi.aspects.apt.QuietlyReturnTypeProcessor, returning RELEASE_6.
[INFO] [WARNING] diagnostic: warning: Supported source version 'RELEASE_6' from annotation processor 'com.jcabi.aspects.apt.QuietlyReturnTypeProcessor' less than -source '1.8'
[INFO] [ERROR] diagnostic: error: Method 'QuietlyNonVoid.foo' annotated with @Quietly does not return void
[INFO] [WARNING] diagnostic: warning: No SupportedSourceVersion annotation found on com.jcabi.aspects.apt.AsyncReturnTypeProcessor, returning RELEASE_6.
[INFO] [WARNING] diagnostic: warning: Supported source version 'RELEASE_6' from annotation processor 'com.jcabi.aspects.apt.AsyncReturnTypeProcessor' less than -source '1.8'
[INFO] [ERROR] diagnostic: error: Method 'AsyncInvalid.foo' annotated with @Async does not return void or Future
[INFO] [ERROR] error on execute: error during compilation
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD FAILURE
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Total time: 20.851 s
[INFO] [INFO] Finished at: 2015-09-12T14:16:30+03:00
[INFO] [INFO] Final Memory: 17M/76M
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [ERROR] Failed to execute goal org.bsc.maven:maven-processor-plugin:2.2.4:process (process) on project jcabi-test: Error executing: error during compilation -> [Help 1]
[INFO] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.bsc.maven:maven-processor-plugin:2.2.4:process (process) on project jcabi-test: Error executing
[INFO]  at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:216)
[INFO]  at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
[INFO]  at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
[INFO]  at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
[INFO]  at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
[INFO]  at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
[INFO]  at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
[INFO]  at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
[INFO]  at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
[INFO]  at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
[INFO]  at org.apache.maven.cli.MavenCli.execute(MavenCli.java:862)
[INFO]  at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:286)
[INFO]  at org.apache.maven.cli.MavenCli.main(MavenCli.java:197)
[INFO]  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO]  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO]  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO]  at java.lang.reflect.Method.invoke(Method.java:483)
[INFO]  at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
[INFO]  at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
[INFO]  at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
[INFO]  at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[INFO] Caused by: org.apache.maven.plugin.MojoExecutionException: Error executing
[INFO]  at org.bsc.maven.plugin.processor.AbstractAnnotationProcessorMojo.execute(AbstractAnnotationProcessorMojo.java:292)
[INFO]  at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
[INFO]  at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
[INFO]  ... 20 more
[INFO] Caused by: java.lang.Exception: error during compilation
[INFO]  at org.bsc.maven.plugin.processor.AbstractAnnotationProcessorMojo.executeWithExceptionsHandled(AbstractAnnotationProcessorMojo.java:548)
[INFO]  at org.bsc.maven.plugin.processor.AbstractAnnotationProcessorMojo.execute(AbstractAnnotationProcessorMojo.java:285)
[INFO]  ... 22 more
[INFO] [ERROR] 
[INFO] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
[INFO] [ERROR] 
[INFO] [ERROR] For more information about the errors and possible solutions, please read the following articles:
[INFO] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[INFO] Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8
[INFO] run script verify.groovy
[INFO] Running post-build script: E:\Projects\oDesk\Teamed.io\jcabi\jcabi-aspects\jcabi-aspects\target\it\processor\verify.groovy
[INFO] Finished post-build script: E:\Projects\oDesk\Teamed.io\jcabi\jcabi-aspects\jcabi-aspects\target\it\processor\verify.groovy
[INFO] ..SUCCESS (23.4 s)
[INFO] Building: weaving\pom.xml
[INFO] [INFO] Error stacktraces are turned on.
[INFO] [INFO] Scanning for projects...
[INFO] [INFO]                                                                         
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Building test 1.0
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Downloading: file:///C:/Users/YamStranger/.m2/repository/org/projectlombok/lombok/0.11.6/lombok-0.11.6.pom
[INFO] [INFO] Downloading: https://oss.sonatype.org/content/groups/public/org/projectlombok/lombok/0.11.6/lombok-0.11.6.pom
[INFO] [INFO] Downloaded: https://oss.sonatype.org/content/groups/public/org/projectlombok/lombok/0.11.6/lombok-0.11.6.pom (2 KB at 1.1 KB/sec)
[INFO] [INFO] Downloading: file:///C:/Users/YamStranger/.m2/repository/org/projectlombok/lombok/0.11.6/lombok-0.11.6.jar
[INFO] [INFO] Downloading: https://oss.sonatype.org/content/groups/public/org/projectlombok/lombok/0.11.6/lombok-0.11.6.jar
[INFO] [INFO] Downloaded: https://oss.sonatype.org/content/groups/public/org/projectlombok/lombok/0.11.6/lombok-0.11.6.jar (1780 KB at 586.3 KB/sec)
[INFO] [INFO] 
[INFO] [INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ jcabi-test ---
[INFO] [INFO] 
[INFO] [INFO] --- buildnumber-maven-plugin:1.2:create (jcabi-build-number) @ jcabi-test ---
[INFO] [INFO] Checking for local modifications: skipped.
[INFO] [INFO] Updating project files from SCM: skipped.
[INFO] [INFO] ShortRevision tag detected. The value is '8'.
[INFO] [INFO] Executing: cmd.exe /X /C "git rev-parse --verify --short=8 HEAD"
[INFO] [INFO] Working directory: E:\Projects\oDesk\Teamed.io\jcabi\jcabi-aspects\jcabi-aspects\target\it\weaving
[INFO] [INFO] Storing buildNumber: ab0bf9a8 at timestamp: 1442056600765
[INFO] [INFO] ShortRevision tag detected. The value is '8'.
[INFO] [INFO] Executing: cmd.exe /X /C "git rev-parse --verify --short=8 HEAD"
[INFO] [INFO] Working directory: E:\Projects\oDesk\Teamed.io\jcabi\jcabi-aspects\jcabi-aspects\target\it\weaving
[INFO] [INFO] Storing buildScmBranch: UNKNOWN
[INFO] [INFO] 
[INFO] [INFO] --- maven-antrun-plugin:1.7:run (jcabi-prepare-ajc) @ jcabi-test ---
[INFO] [INFO] Executing tasks
[INFO] 
[INFO] main:
[INFO] [INFO] Executed tasks
[INFO] [INFO] 
[INFO] [INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ jcabi-test ---
[INFO] [INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] [INFO] skip non existing resourceDirectory E:\Projects\oDesk\Teamed.io\jcabi\jcabi-aspects\jcabi-aspects\target\it\weaving\src\main\resources
[INFO] [INFO] 
[INFO] [INFO] --- maven-compiler-plugin:3.0:compile (default-compile) @ jcabi-test ---
[INFO] [INFO] Changes detected - recompiling the module!
[INFO] [INFO] Compiling 4 source files to E:\Projects\oDesk\Teamed.io\jcabi\jcabi-aspects\jcabi-aspects\target\it\weaving\target\classes
[INFO] [INFO] -------------------------------------------------------------
[INFO] [WARNING] COMPILATION WARNING : 
[INFO] [INFO] -------------------------------------------------------------
[INFO] [WARNING] /E:/Projects/oDesk/Teamed.io/jcabi/jcabi-aspects/jcabi-aspects/target/it/weaving/src/main/java/com/jcabi/Document.java:[41,14] Can't initialize javac processor due to (most likely) a class loader problem: java.lang.NoClassDefFoundError: com/sun/tools/javac/code/TypeTags
[INFO]      at lombok.javac.handlers.HandleGetter.<clinit>(HandleGetter.java:291)
[INFO]      at java.lang.Class.forName0(Native Method)
[INFO]      at java.lang.Class.forName(Class.java:340)
[INFO]      at lombok.core.SpiLoadUtil$1$1.next(SpiLoadUtil.java:111)
[INFO]      at lombok.javac.HandlerLibrary.loadAnnotationHandlers(HandlerLibrary.java:167)
[INFO]      at lombok.javac.HandlerLibrary.load(HandlerLibrary.java:152)
[INFO]      at lombok.javac.JavacTransformer.<init>(JavacTransformer.java:43)
[INFO]      at lombok.javac.apt.Processor.init(Processor.java:90)
[INFO]      at lombok.core.AnnotationProcessor$JavacDescriptor.want(AnnotationProcessor.java:89)
[INFO]      at lombok.core.AnnotationProcessor.init(AnnotationProcessor.java:143)
[INFO]      at com.sun.tools.javac.processing.JavacProcessingEnvironment$ProcessorState.<init>(JavacProcessingEnvironment.java:500)
[INFO]      at com.sun.tools.javac.processing.JavacProcessingEnvironment$DiscoveredProcessors$ProcessorStateIterator.next(JavacProcessingEnvironment.java:597)
[INFO]      at com.sun.tools.javac.processing.JavacProcessingEnvironment.discoverAndRunProcs(JavacProcessingEnvironment.java:690)
[INFO]      at com.sun.tools.javac.processing.JavacProcessingEnvironment.access$1800(JavacProcessingEnvironment.java:91)
[INFO]      at com.sun.tools.javac.processing.JavacProcessingEnvironment$Round.run(JavacProcessingEnvironment.java:1035)
[INFO]      at com.sun.tools.javac.processing.JavacProcessingEnvironment.doProcessing(JavacProcessingEnvironment.java:1176)
[INFO]      at com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1173)
[INFO]      at com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:859)
[INFO]      at com.sun.tools.javac.main.Main.compile(Main.java:523)
[INFO]      at com.sun.tools.javac.api.JavacTaskImpl.doCall(JavacTaskImpl.java:129)
[INFO]      at com.sun.tools.javac.api.JavacTaskImpl.call(JavacTaskImpl.java:138)
[INFO]      at org.codehaus.plexus.compiler.javac.JavaxToolsCompiler.compileInProcess(JavaxToolsCompiler.java:126)
[INFO]      at org.codehaus.plexus.compiler.javac.JavacCompiler.performCompile(JavacCompiler.java:170)
[INFO]      at org.apache.maven.plugin.compiler.AbstractCompilerMojo.execute(AbstractCompilerMojo.java:684)
[INFO]      at org.apache.maven.plugin.compiler.CompilerMojo.execute(CompilerMojo.java:118)
[INFO]      at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
[INFO]      at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
[INFO]      at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
[INFO]      at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
[INFO]      at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
[INFO]      at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
[INFO]      at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
[INFO]      at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
[INFO]      at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
[INFO]      at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
[INFO]      at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
[INFO]      at org.apache.maven.cli.MavenCli.execute(MavenCli.java:862)
[INFO]      at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:286)
[INFO]      at org.apache.maven.cli.MavenCli.main(MavenCli.java:197)
[INFO]      at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO]      at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO]      at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO]      at java.lang.reflect.Method.invoke(Method.java:483)
[INFO]      at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
[INFO]      at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
[INFO]      at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
[INFO]      at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[INFO]   Caused by: java.lang.ClassNotFoundException: com.sun.tools.javac.code.TypeTags
[INFO]      at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
[INFO]      at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
[INFO]      at java.security.AccessController.doPrivileged(Native Method)
[INFO]      at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
[INFO]      at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[INFO]      at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[INFO]      ... 47 more
[INFO] 
[INFO] [INFO] 1 warning
[INFO] [INFO] -------------------------------------------------------------
[INFO] [INFO] -------------------------------------------------------------
[INFO] [ERROR] COMPILATION ERROR : 
[INFO] [INFO] -------------------------------------------------------------
[INFO] [ERROR] warnings found and -Werror specified
[INFO] [INFO] 1 error
[INFO] [INFO] -------------------------------------------------------------
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD FAILURE
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Total time: 9.589 s
[INFO] [INFO] Finished at: 2015-09-12T14:16:43+03:00
[INFO] [INFO] Final Memory: 20M/75M
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.0:compile (default-compile) on project jcabi-test: Compilation failure
[INFO] [ERROR] warnings found and -Werror specified
[INFO] [ERROR] -> [Help 1]
[INFO] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.0:compile (default-compile) on project jcabi-test: Compilation failure
[INFO] warnings found and -Werror specified
[INFO] 
[INFO]  at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
[INFO]  at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
[INFO]  at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
[INFO]  at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
[INFO]  at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
[INFO]  at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
[INFO]  at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
[INFO]  at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
[INFO]  at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
[INFO]  at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
[INFO]  at org.apache.maven.cli.MavenCli.execute(MavenCli.java:862)
[INFO]  at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:286)
[INFO]  at org.apache.maven.cli.MavenCli.main(MavenCli.java:197)
[INFO]  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO]  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO]  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO]  at java.lang.reflect.Method.invoke(Method.java:483)
[INFO]  at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
[INFO]  at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
[INFO]  at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
[INFO]  at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[INFO] Caused by: org.apache.maven.plugin.compiler.CompilationFailureException: Compilation failure
[INFO] warnings found and -Werror specified
[INFO] 
[INFO]  at org.apache.maven.plugin.compiler.AbstractCompilerMojo.execute(AbstractCompilerMojo.java:745)
[INFO]  at org.apache.maven.plugin.compiler.CompilerMojo.execute(CompilerMojo.java:118)
[INFO]  at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
[INFO]  at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
[INFO]  ... 20 more
[INFO] [ERROR] 
[INFO] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
[INFO] [ERROR] 
[INFO] [ERROR] For more information about the errors and possible solutions, please read the following articles:
[INFO] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[INFO] Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8
[INFO] ..FAILED (12.1 s)
[INFO]   The build exited with code 1. See E:\Projects\oDesk\Teamed.io\jcabi\jcabi-aspects\jcabi-aspects\target\it\weaving\build.log for details.
[INFO] -------------------------------------------------
[INFO] Build Summary:
[INFO]   Passed: 2, Failed: 1, Errors: 0, Skipped: 0
[INFO] -------------------------------------------------
[ERROR] The following builds failed:
[ERROR] *  weaving\pom.xml
[INFO] -------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 03:38 min
[INFO] Finished at: 2015-09-12T14:16:43+03:00
[INFO] Final Memory: 35M/141M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-invoker-plugin:1.9:run (default) on project jcabi-aspects: 1 build failed. See console output above for details. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

**Comment by YamStranger** [comment]
If I exclude it from build (integration's tests) - build success, under Java 7

**Comment by dmarkov** [comment]
@yegor256 dispatch this issue please

**Comment by yegor256** [comment]
@YamStranger how can I reproduce this?

**Comment by YamStranger** [comment]
@yegor256 I cloned repo, and run build. Integration test was fallen with "jcabi-aspects/jcabi-aspects/target/it/weaving/src/main/java/com/jcabi/Document.java:[41,14] Can't initialize javac processor due to (most likely) a class loader problem: java.lang.NoClassDefFoundError: com/sun/tools/javac/code/TypeTags". So All my fixes and pushes to this repo was done without integration tests, but with "just tests". Funny story that on your CI all works correctly. 
Java 7. Windows. Nothing special - all other projects work and build

**Comment by yegor256** [comment]
@YamStranger so, it's just Windows+Java7 and build fails?

**Comment by YamStranger** [comment]
@yegor256 yes

**Comment by yegor256** [comment]
@YamStranger what about this: https://ci.appveyor.com/project/yegor256/jcabi-aspects/branch/master

It's Windows+Java7

**Comment by YamStranger** [comment]
@yegor256 I removed integration tests in my local repository, I push all changes only after successful unit tests, but without checking integration tests.

**Comment by yegor256** [comment]
@YamStranger and? I didn't get the point... Appveyor says that the build is clean, on Windows+Java7

**Comment by YamStranger** [comment]
@yegor256 I read this https://github.com/jcabi/jcabi-aspects, and followed by instructions.
Here is full log of  failed build (https://www.dropbox.com/s/b4nq7dxzts10gcd/out.lst?dl=0 ), and it creating recorded to this video(https://youtu.be/X4lrxxNmuCU ). 

I think, that when build of repo is failed after following all instructions - it is issue, that why I reported it. I think, that Appveyor has some configuration that not explained in README.md

Also my mvn and java version: 
$ java -version
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8
java version "1.7.0_65"
Java(TM) SE Runtime Environment (build 1.7.0_65-b20)
Java HotSpot(TM) Client VM (build 24.65-b04, mixed mode, sharing)

$ mvn -version
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8
Apache Maven 3.2.1 (ea8b2b07643dbb1b84b6d16e1f08391b666bc1e9; 2014-02-14T19:37:5
2+02:00)
Maven home: c:\BuildSystems\Maven\apache-maven
Java version: 1.8.0_05, vendor: Oracle Corporation
Java home: c:\Java\jdk8.5\jre
Default locale: en_US, platform encoding: UTF-8
OS name: "windows 7", version: "6.1", arch: "amd64", family: "dos"

**Comment by yegor256** [comment]
@YamStranger I think that the problem is that you have JRE, not JDK. Try to install JDK and run the build again

**Comment by YamStranger** [comment]
@yegor256 unfortunately no. If I have only jre - I could not build any project, but I can. And I can build this project without "it" folder (under "test" we have a lot of tests, and they execute successfully ). Problem in some enother place

**Comment by YamStranger** [comment]
@yegor256 I retried with new JDK (why not). Here is result of failed build: https://www.dropbox.com/s/u8n3spyiu8jic8m/out?dl=0 

Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8
Apache Maven 3.2.1 (ea8b2b07643dbb1b84b6d16e1f08391b666bc1e9; 2014-02-14T19:37:5
2+02:00)
Maven home: c:\BuildSystems\Maven\apache-maven\bin\..
Java version: 1.7.0_67, vendor: Oracle Corporation
Java home: c:\Java\jdk1.7.0_67\jre
Default locale: en_US, platform encoding: UTF-8
OS name: "windows 7", version: "6.1", arch: "amd64", family: "windows"

**Comment by yegor256** [comment]
@YamStranger still looks like JRE to me:

```
Java home: c:\Java\jdk1.7.0_67\jre
```

**Comment by YamStranger** [comment]
@yegor256 Here is your link: https://ci.appveyor.com/project/yegor256/jcabi-aspects/branch/master

line 21:
Java home: C:\Program Files\Java\jdk1.7.0\jre

It is normal for mvn -version output. It is not jre, it is jdk.

**Comment by yegor256** [comment]
@YamStranger this is what is happening on your machine:

```
[INFO] [WARNING] /E:/Projects/oDesk/Teamed.io/video/jcabi-aspects/target/it/weaving/src/main/java/com/jcabi/StaticPage.java:[40,14] Can't initialize javac processor due to (most likely) a class loader problem: java.lang.NoClassDefFoundError: com/sun/tools/javac/code/TypeTags
[INFO]   	at lombok.javac.handlers.HandleGetter.<clinit>(HandleGetter.java:291)
[INFO]   	at java.lang.Class.forName0(Native Method)
[INFO]   	at java.lang.Class.forName(Class.java:340)
[INFO]   	at lombok.core.SpiLoadUtil$1$1.next(SpiLoadUtil.java:111)
[INFO]   	at lombok.javac.HandlerLibrary.loadAnnotationHandlers(HandlerLibrary.java:167)
[INFO]   	at lombok.javac.HandlerLibrary.load(HandlerLibrary.java:152)
[INFO]   	at lombok.javac.JavacTransformer.<init>(JavacTransformer.java:43)
[INFO]   	at lombok.javac.apt.Processor.init(Processor.java:90)
[INFO]   	at lombok.core.AnnotationProcessor$JavacDescriptor.want(AnnotationProcessor.java:89)
[INFO]   	at lombok.core.AnnotationProcessor.init(AnnotationProcessor.java:143)
[INFO]   	at com.sun.tools.javac.processing.JavacProcessingEnvironment$ProcessorState.<init>(JavacProcessingEnvironment.java:500)
[INFO]   	at com.sun.tools.javac.processing.JavacProcessingEnvironment$DiscoveredProcessors$ProcessorStateIterator.next(JavacProcessingEnvironment.java:597)
[INFO]   	at com.sun.tools.javac.processing.JavacProcessingEnvironment.discoverAndRunProcs(JavacProcessingEnvironment.java:690)
[INFO]   	at com.sun.tools.javac.processing.JavacProcessingEnvironment.access$1800(JavacProcessingEnvironment.java:91)
[INFO]   	at com.sun.tools.javac.processing.JavacProcessingEnvironment$Round.run(JavacProcessingEnvironment.java:1035)
[INFO]   	at com.sun.tools.javac.processing.JavacProcessingEnvironment.doProcessing(JavacProcessingEnvironment.java:1176)
[INFO]   	at com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1173)
[INFO]   	at com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:859)
[INFO]   	at com.sun.tools.javac.main.Main.compile(Main.java:523)
[INFO]   	at com.sun.tools.javac.api.JavacTaskImpl.doCall(JavacTaskImpl.java:129)
[INFO]   	at com.sun.tools.javac.api.JavacTaskImpl.call(JavacTaskImpl.java:138)
[INFO]   	at org.codehaus.plexus.compiler.javac.JavaxToolsCompiler.compileInProcess(JavaxToolsCompiler.java:126)
[INFO]   	at org.codehaus.plexus.compiler.javac.JavacCompiler.performCompile(JavacCompiler.java:170)
[INFO]   	at org.apache.maven.plugin.compiler.AbstractCompilerMojo.execute(AbstractCompilerMojo.java:684)
[INFO]   	at org.apache.maven.plugin.compiler.CompilerMojo.execute(CompilerMojo.java:118)
[INFO]   	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:133)
[INFO]   	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
[INFO]   	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
[INFO]   	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
[INFO]   	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:108)
[INFO]   	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:76)
[INFO]   	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
[INFO]   	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:116)
[INFO]   	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:361)
[INFO]   	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:155)
[INFO]   	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:584)
[INFO]   	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:213)
[INFO]   	at org.apache.maven.cli.MavenCli.main(MavenCli.java:157)
[INFO]   	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[INFO]   	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[INFO]   	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[INFO]   	at java.lang.reflect.Method.invoke(Method.java:483)
[INFO]   	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
[INFO]   	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
[INFO]   	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
[INFO]   	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[INFO]   Caused by: java.lang.ClassNotFoundException: com.sun.tools.javac.code.TypeTags
[INFO]   	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
[INFO]   	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
[INFO]   	at java.security.AccessController.doPrivileged(Native Method)
[INFO]   	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
[INFO]   	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[INFO]   	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[INFO]   	... 46 more
```

Definitely something is wrong with the JDK... I'm not sure why. And the problem is that we won't be able to fix it, since we can't reproduce it. No matter who we assign this task to will say the same - "everything works fine for me". The best you can do is to try to find what is the problem and suggest a solution here. Than, we can accept this bug.

**Comment by YamStranger** [comment]
@yegor256 Thanks, I will try. Now I have not idea what is the reason of such behavior

**Comment by sfat** [comment]
@yegor256 I've managed to reproduce the bug locally with 
```
Apache Maven 3.2.5 (12a6b3acb947671f09b81f49094c53f426d8cea1; 2014-12-14T19:29:23+02:00)
Maven home: /usr/local/Cellar/maven32/3.2.5/libexec
Java version: 1.8.0_66, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_66.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.11.1", arch: "x86_64", family: "mac"
```
On jdk1.7 it works fine on Mac OS X.
The problem on jdk1.8 it's because of this https://github.com/rzwitserloot/lombok/issues/524
From the looks of it, src/it/weaving is using as parent jcabi-parent, which has lombok version 0.11.6 and the bug was fixed afterwards (1.16 +).
I've managed to make the fix upgrading jcabi-parent to 0.37.1.
Did a regression with 1.7 and 1.8 and worked fine.
Not sure what's the problem with @YamStranger situation, but I think it's similar.

**Comment by yegor256** [comment]
@sfat but we're using jcabi-parent 0.37.1 in master now... can you submit a pull request with your changes?
